microcosm_postgres/metrics.py

```diff
@@ -0,0 +1,84 @@
+from microcosm.api import defaults, typed
+from microcosm.config.types import boolean
+from microcosm.errors import NotBoundError
+from microcosm_logging.timing import elapsed_time
+
+
+@defaults(
+    enabled=typed(boolean, default_value=True)
+)
+class PostgresStoreMetrics:
+    """
+    Send metrics regarding the postgres store
+
+    """
+
+    def __init__(self, graph):
+        self.metrics = self.get_metrics(graph)
+        self.enabled = bool(
+            self.metrics
+            and self.metrics.host != "localhost"
```

Comment:
This is probably cargo-culted, and should be removed wholesale. It's annoying for when you want to test stats against a local exporter.

Reply:
Yeah, I do not mind taking this out. Will do.